policyengine_canada/parameters/gov/provinces/nu/tax/income/credits/single_status_credit/amount.yaml

```diff
@@ -0,0 +1,9 @@
+description: Nunavut provides this amount to person with single status
+values:
+  2022-01-01: 16_000
+metadata:
+  unit: currency-CAD
+  label: Nunavut single status credit amount
+  reference:
+  - title: Nunavut Income Tax Act Part I Division B Subvision c 2.16
+    href: https://www.nunavutlegislation.ca/en/consolidated-law/income-tax-act-consolidation
```

Comment:
could you add the 2022 tax form as well to see if amount adjusted with inflation

Reply:
I don't know why the other reference is now showing up here but the updated amount is inflation-adjusted.

Reply:
yeah. That's interesting. In codespace, i could see the other reference (2022 tax form) but i could not see here. I recommit that pull request. Idk what's going on here.

Reply:
different branch?